zeitschrift-fur-medienwissenschaft.csl

```diff
@@ -20,493 +14,297 @@
     <category field="humanities"/>
     <issn>1869-1722</issn>
     <summary>Style for zfm. Zeitschrift für Medienwissenschaft. Citations in notes; no bibliography.</summary>
-    <updated>2012-09-27T22:06:38+00:00</updated>
+    <updated>2014-09-13T22:01:31+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
-  <!--
-================================================================================
-   TODO
-================================================================================
--->
-  <!--
-================================================================================
-   TERMS
-================================================================================
--->
+  <!-- HINWEISE
+       ========
+       - Bei Zeitschriftenartikeln (article-journal, article-magazine) muss
+         der Verlagsort (publisher-place) ins Zotero-Feld "Extra" eingetragen
+         werden, und zwar nach dem Muster "{:publisher-place: Drexel Hill,
+         Pennsylvania}".
+       - Bei Filmen (motion_picture) muss das Land ins Zotero-Feld "Extra"
+	 eingetragen werden, und zwar dem Muster "{:publisher-place: USA}".
+       - Das Originalpublikationsjahr muss ins Zotero-Feld "Extra" eingetragen
+         werden, und zwar nach dem Muster "{:original-date: 1964}".
+       - Bei Reihen (series) muss die Abkürzung "Bd. " als Präfix für die
+	 "Nummer der Reihe" ins Zotero-Feld "Nummer" eingetragen werden (nach
+	 dem Muster "Bd. 7").
```

Comment:
If the style sheet or common practice is forcing to use different prefixes depending on the series, then you will need to input that information and cannot use CSL-term for that. I suggest to input all information actual volume number and prefix like "Bd. " in the series-number field in zotero, which will then land in the collection-number field in CSL. @adam3smith : Okay? However, my guess is that the different punctation would better be tested on itemType (article or book). Not?